### ISSUE TITLE
Infer domain and time from ABI file?

### ISSUE THREAD

**Issue by gerritholl** [deeplycloudy/glmtools#74]
This is either a question or a feature proposal, depending on whether it's already possible.  I imagine a rather common use case to be that the user wishes to aggregate GLM flashes over the domain and time coverage corresponding to the spatial coverage and interval between two ABI files.  For example, consider:

```
OR_ABI-L1b-RadM1-M6C10_G16_s20201041800242_e20201041800311_c20201041800343.nc
OR_ABI-L1b-RadM1-M6C10_G16_s20201041801213_e20201041801282_c20201041801312.nc
OR_ABI-L1b-RadM1-M6C10_G16_s20201041802213_e20201041802282_c20201041802328.nc
OR_ABI-L1b-RadM1-M6C10_G16_s20201041803213_e20201041803282_c20201041803323.nc
OR_ABI-L1b-RadM1-M6C10_G16_s20201041804213_e20201041804282_c20201041804320.nc
```

then a user processing these alongside GLM L2 data may desire to:

- infer position/sector (spatial domain) from the file contents,
- aggregate GLM flashes according to the time between ABI files, in this case 18:00:24.2/18:01:21.3, 18:01:21.3/18:02:21.3, etc.  (I thought about the actual start/end scan time in ABI, such as 18:00:24.2/18:00:31.1, but that's too short and not useful)

Is either of that functionality currently possible with glmtools, or otherwise desirable?

**Comment by deeplycloudy** [comment]
Yes, it's possible to pass an arbitrary custom start and end time to the gridding system (--start and --end in `make_glm_grids.py`), and if you have the x,y fixed grid coords from ABI data then it should be straightforward to use those to drive the gridding process. `make_glm_grids.py` also already manages the generation of a conus and full disk grids that match ABI, as well as ABI mesoscale domains given a specified center point. When you say `--domain=conus` it generates 1D arrays of x,y fixed grid coordinates that are used as the target grid spec. If you already have explicit x,y from an ABI file those could be used directly, too. So, at first glance, this looks like a straightforward and welcome modification to `make_glm_grids.py`.
